docs/simulator.md

```diff
@@ -41,6 +41,42 @@ the `plugins` block of your app's `build.gradle`/`build.gradle.kts` file:
     ```
     ///
 
+To enable the simulator to work with JDK 23 and newer, you must add the following extra jvmArgs to your unit test configuration:
+
+!!! snippet "simulator-jvmArgs-configurations"
+
+    /// tab | Groovy
+    ```groovy
+    tasks.withType(Test).configureEach {
+        it.jvmArgs(
+            "--add-exports=java.desktop/sun.awt=ALL-UNNAMED",
+            "--add-exports=java.base/sun.reflect.misc=ALL-UNNAMED",
+            "--add-exports=java.base/sun.security.action=ALL-UNNAMED",
+            "--add-exports=java.desktop/sun.swing=ALL-UNNAMED",
+            "--add-exports=java.desktop/sun.lwawt.macosx=ALL-UNNAMED"
+        )
+    }
+    ```
+    ///
+
+    /// tab | Kotlin
+    ```kotlin
+    testOptions {
```

Comment:
Why not use the `tasks.withType(Test).configureEach` here as well?

Reply:
Doesn't it work to do the following in Groovy?
```groovy
testOptions {
    unitTests.all { test ->
        test.jvmArgs = ...
    }
}
```

Reply:
When I have time, I can give it a test.